src/visualize.jl

```diff
@@ -34,7 +34,7 @@ function plot_Ws(
         a.imshow(transpose(W[:, idx, k]), aspect="auto")
     end
 
-    if ~isnothing(trueW)
+    if !(trueW === nothing)
```

Comment:
Why do we use a triple `===` here?

Reply:
Apparently it is faster in some situations so I went with the "proper" convention. It doesn't matter here though.